packages/react/src/Avatar/Avatar.tsx

```diff
@@ -9,7 +9,6 @@ import { forwardRef } from 'react'
 import type { ForwardedRef, HTMLAttributes } from 'react'
 import { Icon } from '../Icon'
 import { Image } from '../Image'
-import { VisuallyHidden } from '../VisuallyHidden'
```

Comment:
Is the component itself going to be removed too?

Reply:
Probably, yeah